samples/ipc_call_interface/common/cipher_1_model.h

```diff
@@ -7,11 +7,12 @@
  * modified.
  ********************************************************************/
 
-#ifndef CIPHER_1_INTERFACE_H
-#define CIPHER_1_INTERFACE_H
+#ifndef CIPHER_1_MODEL_H
```

Comment:
wasn't the plan to rename this to keyvault?

Reply:
Not in this sample, let's keep it a simple cipher.